### PR TITLE
added hello world title with js

### PR DESCRIPTION
**Pull request by darioalbanesi**
this here is to just add "Hello World" as a title on index.html

### CODE REVIEW

File: js/helloworld/helloworld.js [hunk 1]
```diff
@@ -1 +1 @@
-alert('foo');
+$('.js-title').text('Hello World');
```
Comment (vxsx) [inline js/helloworld/helloworld.js]: while this works, you should always keep in mind that a lot of times you don't have control over you javascript files (especially if it's a library people will use), for example if this file will be concatenated with others and placed in the head it won't work. And that's assuming you have some control over your dependencies.

So yeah, you should always do `$(function() { ... })` or equivalent `$(document).ready(function () { ... })`

File: index.html [hunk 2]
```diff
@@ -21,9 +21,14 @@
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
 
-        <script src="js/vendor/jquery-1.10.2.min.js"></script>
-        <script src="js/helloworld/helloworld.js"></script>
+        <div class="container">
+            <div class="title">
+                <h2 class="js-title"></h2>
+            </div>
+        </div>
```
Comment (FinalAngel) [inline index.html]: to many empty lines?

File: index.html [hunk 3]
```diff
@@ -21,9 +21,14 @@
             <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
         <![endif]-->
 
-        <script src="js/vendor/jquery-1.10.2.min.js"></script>
-        <script src="js/helloworld/helloworld.js"></script>
+        <div class="container">
+            <div class="title">
```
Comment (FinalAngel) [inline index.html]: though its a very small site, "title" is very generic, what is it? headline-title, blog-title, container-title?

**Comment by vxsx** [comment]
LGTM :)

**Comment by FinalAngel** [comment]
@darioalbanesi merge merge ;)

**Pull request merged**